plugin/trino-mysql/src/main/java/io/trino/plugin/mysql/MySqlClient.java

```diff
@@ -470,6 +531,35 @@ public Optional<ColumnMapping> toColumnMapping(ConnectorSession session, Connect
         return Optional.empty();
     }
 
+    private static ColumnMapping mySqlDefaultVarcharColumnMapping(int columnSize, boolean isRemoteCaseSensitive)
```

Comment:
Let's use `CaseSensitivity` instead of `boolean`, or even `Optional<CaseSensitivity>`

Reply:
refactored